### PR TITLE
Refine init command

### LINKED ISSUE #845: Refine the `protostar init` command
- don't ask about the lib directory (a user can always change the directory later)
- allow providing the project name through cli to skip the prompt

### PR DESCRIPTION
**Pull request by Arcticae**
closes #845

### CODE REVIEW

File: protostar/commands/init/init_command.py [hunk 1]
```diff
@@ -42,16 +42,30 @@ def arguments(self) -> List[Command.Argument]:
...
                 description="Adapt current directory to a Protostar project.",
                 type="bool",
-            )
+            ),
+            Command.Argument(
+                name="name",
+                description="Name of the directory a new project will be placed in. "
+                "Ignored when `--existing` is passed.",
+                type="str",
+            ),
```
Comment (kasperski95) [inline protostar/commands/init/init_command.py:51]: [Functionality] I'd make it optional position argument.
`protostar init my-new-project`

File: protostar/commands/init/project_creator/_project_creator.py [hunk 2]
```diff
@@ -28,14 +28,20 @@ def copy_template(self, template_name: Literal["default"], project_root_path: Pa
...
-            protostar_project=ProtostarProjectSection(libs_relative_path=libs_path),
+            protostar_project=ProtostarProjectSection(
+                libs_relative_path=Path(self.default_lib_dirname),
+            ),
             protostar_contracts=ProtostarContractsSection.get_default(),
         )
 
+    @property
+    def default_lib_dirname(self) -> str:
+        return "lib"
```
Comment (kasperski95) [inline protostar/commands/init/project_creator/_project_creator.py:43]: [Complexity] This property could be defined in the constructor.

**Review by kasperski95 — CHANGES_REQUESTED** [review]
Documentation is not updated.

File: protostar/protostar_toml/io/protostar_toml_writer.py [hunk 3]
```diff
@@ -29,6 +29,5 @@ def save(
 
         for section in sections:
             result[f"protostar.{section.get_section_name()}"] = section.to_dict()
-
```
Comment (mkaput) [inline protostar/protostar_toml/io/protostar_toml_writer.py:32]: change out of scope of this PR

**Review by mkaput — CHANGES_REQUESTED** [review]

File: protostar/commands/init/project_creator/_project_creator.py (hunk 2 above)
Comment (Arcticae) [reply protostar/commands/init/project_creator/_project_creator.py]: done

File: protostar/commands/init/init_command.py (hunk 1 above)
Comment (Arcticae) [reply protostar/commands/init/init_command.py]: done

**Comment by Arcticae** [comment]
@kasperski95 i updated the docs

**Review by kasperski95 — APPROVED** [review]
LGTM

**Review by mkaput — APPROVED** [review]

**Pull request merged**